src/lib/y2storage/encryption.rb

```diff
@@ -57,6 +57,23 @@ class Encryption < BlkDevice
     #   @return [Boolean] whether the device is included in /etc/crypttab
     storage_forward :in_etc_crypttab?
 
+    # @!attribute key_file
+    #   @return [nil] the encryption key file
```

Comment:
`nil`? I guess it returns a String, see https://github.com/openSUSE/libstorage-ng/blob/master/storage/Devices/Encryption.h#L77

Reply:
Good catch! Thanks!